spid_cie_oidc/onboarding/schemas/token_requests.py

```diff
@@ -20,26 +20,26 @@ class TokenAuthnCodeRequest(TokenRequest):
     code_verifier: str
     grant_type: Literal["authorization_code"]
 
-    def example():
-        return TokenAuthnCodeRequest( # noqa: E722
-            client_id = "http://example.com",  # noqa: E722
-            client_assertion = "string.string.string", # noqa: E722
-            client_assertion_type = "urn:ietf:params:oauth:client-assertion-type:jwt-bearer",  # noqa: E722
-            code = "string", # noqa: E722
-            code_verifier = "string", # noqa: E722
-            grant_type = "authorization_code"  # noqa: E722
+    def example():# noqa: F722
+        return TokenAuthnCodeRequest( # noqa: F722
+            client_id = "http://example.com",  # noqa: F722
+            client_assertion = "string.string.string", # noqa: F722
+            client_assertion_type = "urn:ietf:params:oauth:client-assertion-type:jwt-bearer",  # noqa: F722
+            code = "string", # noqa: F722
```

Comment:
In the example stringa let's do something real, example, nonce must be at least 32 digits

The sn example of code challenge ... And so on